.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release notation-azure-kv
+
+on:
+  push:
+    tags:
+      - 'v*'
+
```

Comment:
make sure to define required permissions https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

Reply:
Added permissions section below for release. We should add a PR builder workflow as well but this won't use any tokens.

Reply:
We should change the permission here to `Read repository contents permission` too as the default is read and write.
![Screen Shot 2021-12-14 at 11 15 29 AM](https://user-images.githubusercontent.com/10608509/146064526-6bfdeacf-8e90-483b-89e3-c05b51686414.png)

Reply:
Given that this is creating a release, its needs release permissions is what I observed. If there is another workflow in another repo that works for creating release, could you point me to it and I can make the updates.